central/sac/authorizer/builtin_scoped_authorizer.go

```diff
@@ -25,20 +23,14 @@ var (
 	ErrUnexpectedScopeKey = errors.New("unexpected scope key")
 	// ErrUnknownResource is returned when resource is unknown.
 	ErrUnknownResource = errors.New("unknown resource")
-
-	clusterCache = objectarraycache.NewObjectArrayCache(cacheRefreshPeriod, fetchClustersFromDB)
-)
-
-const (
-	cacheRefreshPeriod = 5 * time.Second
 )
 
 // NewBuiltInScopeChecker returns a new SAC-aware scope checker for the given
 // list of roles.
 func NewBuiltInScopeChecker(ctx context.Context, roles []permissions.ResolvedRole) (sac.ScopeCheckerCore, error) {
 	adminCtx := sac.WithGlobalAccessScopeChecker(ctx, sac.AllowAllAccessScopeChecker())
 
-	clusters, err := fetchClusters(adminCtx)
+	clusters, err := clusterStore.Singleton().GetClusters(adminCtx)
```

Comment:
Similar comment as the other PR, but this will cause multiple copies of the same data right?